client/webserver/site/src/js/wallets.ts

```diff
@@ -149,24 +152,43 @@ export default class WalletsPage extends BasePage {
 
     // Clicking on the available amount on the Send form populates the
     // amount field.
-    Doc.bind(page.sendAvail, 'click', () => {
-      const { wallet: { balance: { available: avail }, traits }, unitInfo: ui } = app().assets[this.selectedAssetID]
-      page.sendAmt.value = String(avail / ui.conventional.conversionFactor)
-      this.showFiatValue(this.selectedAssetID, avail, page.sendValue)
-      // Ensure we don't check subtract checkbox for assets that don't have a
-      // withdraw method.
-      if ((traits & traitWithdrawer) === 0) page.subtractCheckBox.checked = false
-      else page.subtractCheckBox.checked = true
+    Doc.bind(page.walletBal, 'click', () => {
+      const asset = app().assets[this.selectedAssetID]
+      this.populateMaxSend(asset.wallet.balance.available, asset)
```

Comment:
I would think that the click handler for `page.walletBal` would be similar to the handler for `page.maxSend`, considering `traitWithdrawer` and filling with the value in `page.maxSend` _if there is a value there_, otherwise use the value from `asset.wallet.balance.available` like here.

https://user-images.githubusercontent.com/9373513/186438030-5a1ac36c-7cb6-4162-862a-a6ca72c3b399.mp4

It doesn't seem helpful to populate the `sendAmt` box with a value that we can know is not possible to send.